projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -597,7 +599,8 @@ export class RecordSearchComponent implements OnInit, OnChanges, OnDestroy {
           doc_count: aggr[key].doc_count || null,
           value: { buckets: aggr[key].buckets },
           type: aggr[key].type || 'terms',
-          config: aggr[key].config || null
+          config: aggr[key].config || null,
+          name: this._aggregationName(key)
```

Comment:
Could you change this statement as this:

```
name: aggr[key].name || this._aggregationName(key)
```

With this, we can set the name in the backend.